clients/drcachesim/tracer/raw2trace.h

```diff
@@ -1442,6 +1455,75 @@ template <typename T> class trace_converter_t {
         return "";
     }
 
+#ifdef BUILD_PT_POST_PROCESSOR
+#    define PT_DATA_FILE_NAME_SUFFIX ".pt"
+#    define PT_METADAT_FILE_NAME_SUFFIX ".metadata"
+#    define KERNEL_IMAGE_FILE_NAME "kimage"
+#    define KERNEL_IMAGE_METADATA_FILE_NAME "kimage.metadata"
+    std::string
+    append_syscall_pt_trace(void *tls, thread_id_t thread_id, uint64_t syscall_id)
+    {
+        std::string error = "";
+        pt2ir_config_t config = {};
+        config.raw_file_path = impl()->get_syscall_pt_trace_dir(tls) + "/" +
```

Comment:
raw2trace cannot open files, to support use in environments where library-based filesystems are used.  Any files used need to be passed in as istreams or some similar mechanism, or if all the files involved are relatively small we could require that they be copied to local paths where a unix file open will work (we do that today for module_mapper_t binaries, and for the encoding file we pass in a file_t).  But the actual PT trace data is likely going to get large, so I do not think requiring local files is a viable solution.